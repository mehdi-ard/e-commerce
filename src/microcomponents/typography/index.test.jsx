import React from "react";
import renderer from "react-test-renderer";
import Typography from "./";

it("renders default typography", () => {
  expect(
    renderer.create(<Typography>cdjcsdkj</Typography>).toJSON()
  ).toMatchSnapshot();
});

it("renders variant typography", () => {
  expect(
    renderer.create(<Typography variant="h2">cdjcsdkj</Typography>).toJSON()
  ).toMatchSnapshot();
});
