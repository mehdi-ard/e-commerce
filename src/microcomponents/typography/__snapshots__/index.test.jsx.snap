// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default typography 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1"
>
  cdjcsdkj
</p>
`;

exports[`renders variant typography 1`] = `
<h2
  className="MuiTypography-root MuiTypography-h2"
>
  cdjcsdkj
</h2>
`;
